app crash when i crop image

here is my AndroidManifest.xml

```
<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.example.cosin">

    <!-- io.flutter.app.FlutterApplication is an android.app.Application that
         calls FlutterMain.startInitialization(this); in its onCreate method.
         In most cases you can leave this as-is, but you if you want to provide
         additional functionality it is fine to subclass or reimplement
         FlutterApplication and put your custom class here. -->
    <uses-permission android:name="android.permission.INTERNET"/>
    <application android:name="io.flutter.app.FlutterApplication" android:label="cosin" android:icon="@mipmap/ic_launcher">
        <activity android:name=".MainActivity" android:launchMode="singleTop" android:theme="@style/LaunchTheme" android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode" android:hardwareAccelerated="true" android:windowSoftInputMode="adjustResize">
            <!-- This keeps the window background of the activity showing
                 until Flutter renders its first frame. It can be removed if
                 there is no splash screen (such as the default splash screen
                 defined in @style/LaunchTheme). -->
            <meta-data android:name="io.flutter.app.android.SplashScreenUntilFirstFrame" android:value="true" />
            <intent-filter>
                <action android:name="android.intent.action.MAIN"/>
                <category android:name="android.intent.category.LAUNCHER"/>
            </intent-filter>
        </activity>
        <activity android:name="com.yalantis.ucrop.UCropActivity" android:screenOrientation="portrait" android:theme="@style/Theme.AppCompat.Light.NoActionBar"/>
    </application>
</manifest>
```

here is build.gradle
```
def localProperties = new Properties()
def localPropertiesFile = rootProject.file('local.properties')
if (localPropertiesFile.exists()) {
    localPropertiesFile.withReader('UTF-8') { reader ->
        localProperties.load(reader)
    }
}

def flutterRoot = localProperties.getProperty('flutter.sdk')
if (flutterRoot == null) {
    throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
}

def flutterVersionCode = localProperties.getProperty('flutter.versionCode')
if (flutterVersionCode == null) {
    flutterVersionCode = '1'
}

def flutterVersionName = localProperties.getProperty('flutter.versionName')
if (flutterVersionName == null) {
    flutterVersionName = '1.0'
}

apply plugin: 'com.android.application'
apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"

def keystorePropertiesFile = rootProject.file("key.properties")
def keystoreProperties = new Properties()
keystoreProperties.load(new FileInputStream(keystorePropertiesFile))

android {
    compileSdkVersion 28

    lintOptions {
        disable 'InvalidPackage'
    }

    defaultConfig {
        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
        applicationId "com.example.cosin"
        minSdkVersion 16
        targetSdkVersion 28
        versionCode flutterVersionCode.toInteger()
        versionName flutterVersionName
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }

    signingConfigs {
    release {
        keyAlias keystoreProperties['keyAlias']
        keyPassword keystoreProperties['keyPassword']
        storeFile file(keystoreProperties['storeFile'])
        storePassword keystoreProperties['storePassword']
        }
    }

    buildTypes {
        release {
            // TODO: Add your own signing config for the release build.
            // Signing with the debug keys for now, so `flutter run --release` works.
            signingConfig signingConfigs.debug
        }
    }
}

flutter {
    source '../..'
}

dependencies {
    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'com.android.support.test:runner:1.0.2'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
}
```

the error message is 
```
I/zygote64( 1818): Rejecting re-init on previously-failed class java.lang.Class<com.yalantis.ucrop.UCropActivity>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/support/v7/app/AppCompatActivity;
I/zygote64( 1818):   at android.content.Intent com.yalantis.ucrop.UCrop.getIntent(android.content.Context) (UCrop.java:186)
I/zygote64( 1818):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity, int) (UCrop.java:138)
I/zygote64( 1818):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity) (UCrop.java:128)
I/zygote64( 1818):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperDelegate.startCrop(io.flutter.plugin.common.MethodCall, io.flutter.plugin.common.MethodChannel$Result) (ImageCropperDelegate.java:91)
I/zygote64( 1818):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperPlugin.onMethodCall(io.flutter.plugin.common.MethodCall, io.flutter.plugin.common.MethodChannel$Result) (ImageCropperPlugin.java:36)
I/zygote64( 1818):   at void io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(java.nio.ByteBuffer, io.flutter.plugin.common.BinaryMessenger$BinaryReply) (MethodChannel.java:201)
I/zygote64( 1818):   at void io.flutter.view.FlutterNativeView$PlatformMessageHandlerImpl.handleMessageFromDart(java.lang.String, byte[], int) (FlutterNativeView.java:188)
I/zygote64( 1818):   at void io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(java.lang.String, byte[], int) (FlutterJNI.java:202)
I/zygote64( 1818):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64( 1818):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64( 1818):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64( 1818):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6701)
I/zygote64( 1818):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64( 1818):   at void com.android.internal.os.Zygote$MethodAndArgsCaller.run() (Zygote.java:246)
I/zygote64( 1818):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:783)
I/zygote64( 1818): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.app.AppCompatActivity" on path: DexPathList[[zip file "/data/app/com.example.cosin-nHav_I5fp3oS-luRzJkjMQ==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.cosin-nHav_I5fp3oS-luRzJkjMQ==/lib/arm64, /data/app/com.example.cosin-nHav_I5fp3oS-luRzJkjMQ==/base.apk!/lib/arm64-v8a, /system/lib64, /system/vendor/lib64]]
I/zygote64( 1818):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:93)
I/zygote64( 1818):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:379)
I/zygote64( 1818):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
I/zygote64( 1818):   at android.content.Intent com.yalantis.ucrop.UCrop.getIntent(android.content.Context) (UCrop.java:186)
I/zygote64( 1818):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity, int) (UCrop.java:138)
I/zygote64( 1818):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity) (UCrop.java:128)
I/zygote64( 1818):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperDelegate.startCrop(io.flutter.plugin.common.MethodCall, io.flutter.plugin.common.MethodChannel$Result) (ImageCropperDelegate.java:91)
I/zygote64( 1818):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperPlugin.onMethodCall(io.flutter.plugin.common.MethodCall, io.flutter.plugin.common.MethodChannel$Result) (ImageCropperPlugin.java:36)
I/zygote64( 1818):   at void io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(java.nio.ByteBuffer, io.flutter.plugin.common.BinaryMessenger$BinaryReply) (MethodChannel.java:201)
I/zygote64( 1818):   at void io.flutter.view.FlutterNativeView$PlatformMessageHandlerImpl.handleMessageFromDart(java.lang.String, byte[], int) (FlutterNativeView.java:188)
I/zygote64( 1818):   at void io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(java.lang.String, byte[], int) (FlutterJNI.java:202)
I/zygote64( 1818):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64( 1818):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64( 1818):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64( 1818):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6701)
I/zygote64( 1818):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64( 1818):   at void com.android.internal.os.Zygote$MethodAndArgsCaller.run() (Zygote.java:246)
I/zygote64( 1818):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:783)
I/zygote64( 1818):
E/flutter ( 1818): [ERROR:flutter/shell/platform/android/platform_view_android_jni.cc(40)] java.lang.NoClassDefFoundError: Failed resolution of: Lcom/yalantis/ucrop/UCropActivity;
E/flutter ( 1818): 	at com.yalantis.ucrop.UCrop.getIntent(UCrop.java:186)
E/flutter ( 1818): 	at com.yalantis.ucrop.UCrop.start(UCrop.java:138)
E/flutter ( 1818): 	at com.yalantis.ucrop.UCrop.start(UCrop.java:128)
E/flutter ( 1818): 	at vn.hunghd.flutter.plugins.imagecropper.ImageCropperDelegate.startCrop(ImageCropperDelegate.java:91)
E/flutter ( 1818): 	at vn.hunghd.flutter.plugins.imagecropper.ImageCropperPlugin.onMethodCall(ImageCropperPlugin.java:36)
E/flutter ( 1818): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:201)
E/flutter ( 1818): 	at io.flutter.view.FlutterNativeView$PlatformMessageHandlerImpl.handleMessageFromDart(FlutterNativeView.java:188)
E/flutter ( 1818): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:202)
E/flutter ( 1818): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/flutter ( 1818): 	at android.os.MessageQueue.next(MessageQueue.java:331)
E/flutter ( 1818): 	at android.os.Looper.loop(Looper.java:149)
E/flutter ( 1818): 	at android.app.ActivityThread.main(ActivityThread.java:6701)
E/flutter ( 1818): 	at java.lang.reflect.Method.invoke(Native Method)
E/flutter ( 1818): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:246)
E/flutter ( 1818): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:783)
E/flutter ( 1818): Caused by: java.lang.ClassNotFoundException: com.yalantis.ucrop.UCropActivity
E/flutter ( 1818): 	at java.lang.VMClassLoader.findLoadedClass(Native Method)
E/flutter ( 1818): 	at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:738)
E/flutter ( 1818): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:363)
E/flutter ( 1818): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
E/flutter ( 1818): 	... 15 more
```

the code of crash app
```
  var image = await ImagePicker.pickImage(source: ImageSource.gallery);

    File croppedFile = await ImageCropper.cropImage(
      sourcePath: image.path,
      ratioX: 1.0,
      ratioY: 1.0,
      maxWidth: 512,
      maxHeight: 512,
    );

```

[        ] An Observatory debugger and profiler on Mi 4c is available at: http://127.0.0.1:64749/
[        ] For a more detailed help message, press "h". To detach, press "d"; to quit, press "q".
[+5234 ms] V/BoostFramework(26675): BoostFramework() : mPerf = com.qualcomm.qti.Performance@d348b71
[+3126 ms] I/Timeline(26675): Timeline: Activity_launch_request time:74107619 intent:Intent { act=android.intent.action.GET_CONTENT typ=image/* }
[+8223 ms] I/art     (26675): Rejecting re-init on previously-failed class java.lang.Class<com.yalantis.ucrop.UCropActivity>: java.lang.NoClassDefFoundError: Failed resolution of:
Landroid/support/v7/app/AppCompatActivity;
[   +1 ms] I/art     (26675):   at android.content.Intent com.yalantis.ucrop.UCrop.getIntent(android.content.Context) (UCrop.java:186)
[        ] I/art     (26675):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity, int) (UCrop.java:138)
[        ] I/art     (26675):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity) (UCrop.java:128)
[        ] I/art     (26675):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperDelegate.startCrop(io.flutter.plugin.common.MethodCall,
io.flutter.plugin.common.MethodChannel$Result) (ImageCropperDelegate.java:91)
[        ] I/art     (26675):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperPlugin.onMethodCall(io.flutter.plugin.common.MethodCall,
io.flutter.plugin.common.MethodChannel$Result) (ImageCropperPlugin.java:36)
[   +1 ms] I/art     (26675):   at void io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(java.nio.ByteBuffer,
io.flutter.plugin.common.BinaryMessenger$BinaryReply) (MethodChannel.java:201)
[        ] I/art     (26675):   at void io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(java.lang.String, byte[], int) (DartMessenger.java:88)
[        ] I/art     (26675):   at void io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(java.lang.String, byte[], int) (FlutterJNI.java:219)
[        ] I/art     (26675):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
[        ] I/art     (26675):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:323)
[        ] I/art     (26675):   at void android.os.Looper.loop() (Looper.java:142)
[        ] I/art     (26675):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6359)
[        ] I/art     (26675):   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
[        ] I/art     (26675):   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:880)
[        ] I/art     (26675):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:770)
[        ] I/art     (26675): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.app.AppCompatActivity" on path: DexPathList[[zip file
"/data/app/com.example.meccp-2/base.apk"],nativeLibraryDirectories=[/data/app/com.example.meccp-2/lib/arm64, /data/app/com.example.meccp-2/base.apk!/lib/arm64-v8a, /system/lib64,
/vendor/lib64]]
[   +1 ms] I/art     (26675):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
[        ] I/art     (26675):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:380)
[        ] I/art     (26675):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
[        ] I/art     (26675):   at android.content.Intent com.yalantis.ucrop.UCrop.getIntent(android.content.Context) (UCrop.java:186)
[        ] I/art     (26675):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity, int) (UCrop.java:138)
[        ] I/art     (26675):   at void com.yalantis.ucrop.UCrop.start(android.app.Activity) (UCrop.java:128)
[        ] I/art     (26675):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperDelegate.startCrop(io.flutter.plugin.common.MethodCall,
io.flutter.plugin.common.MethodChannel$Result) (ImageCropperDelegate.java:91)
[  +12 ms] I/art     (26675):   at void vn.hunghd.flutter.plugins.imagecropper.ImageCropperPlugin.onMethodCall(io.flutter.plugin.common.MethodCall,
io.flutter.plugin.common.MethodChannel$Result) (ImageCropperPlugin.java:36)
[   +2 ms] I/art     (26675):   at void io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(java.nio.ByteBuffer,
io.flutter.plugin.common.BinaryMessenger$BinaryReply) (MethodChannel.java:201)
[   +1 ms] I/art     (26675):   at void io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(java.lang.String, byte[], int) (DartMessenger.java:88)
[        ] I/art     (26675):   at void io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(java.lang.String, byte[], int) (FlutterJNI.java:219)
[        ] I/art     (26675):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
[        ] I/art     (26675):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:323)
[        ] I/art     (26675):   at void android.os.Looper.loop() (Looper.java:142)
[        ] I/art     (26675):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6359)
[        ] I/art     (26675):   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
[        ] I/art     (26675):   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:880)
[        ] I/art     (26675):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:770)
[        ] I/art     (26675): 
[   +4 ms] E/flutter (26675): [ERROR:flutter/shell/platform/android/platform_view_android_jni.cc(40)] java.lang.NoClassDefFoundError: Failed resolution of:
Lcom/yalantis/ucrop/UCropActivity;
[        ] E/flutter (26675):   at com.yalantis.ucrop.UCrop.getIntent(UCrop.java:186)
[        ] E/flutter (26675):   at com.yalantis.ucrop.UCrop.start(UCrop.java:138)
[   +1 ms] E/flutter (26675):   at com.yalantis.ucrop.UCrop.start(UCrop.java:128)
[        ] E/flutter (26675):   at vn.hunghd.flutter.plugins.imagecropper.ImageCropperDelegate.startCrop(ImageCropperDelegate.java:91)
[        ] E/flutter (26675):   at vn.hunghd.flutter.plugins.imagecropper.ImageCropperPlugin.onMethodCall(ImageCropperPlugin.java:36)
[        ] E/flutter (26675):   at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:201)
[        ] E/flutter (26675):   at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:88)
[        ] E/flutter (26675):   at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:219)
[        ] E/flutter (26675):   at android.os.MessageQueue.nativePollOnce(Native Method)
[        ] E/flutter (26675):   at android.os.MessageQueue.next(MessageQueue.java:323)
[        ] E/flutter (26675):   at android.os.Looper.loop(Looper.java:142)
[        ] E/flutter (26675):   at android.app.ActivityThread.main(ActivityThread.java:6359)
[   +1 ms] E/flutter (26675):   at java.lang.reflect.Method.invoke(Native Method)
[   +1 ms] E/flutter (26675):   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:880)
[        ] E/flutter (26675):   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:770)
[        ] E/flutter (26675): Caused by: java.lang.ClassNotFoundException: com.yalantis.ucrop.UCropActivity
[        ] E/flutter (26675):   at java.lang.VMClassLoader.findLoadedClass(Native Method)
[        ] E/flutter (26675):   at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:742)
[        ] E/flutter (26675):   at java.lang.ClassLoader.loadClass(ClassLoader.java:362)
[        ] E/flutter (26675):   at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
[        ] E/flutter (26675):   ... 15 more
[        ] E/flutter (26675): Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/support/v7/app/AppCompatActivity;
[        ] E/flutter (26675):   ... 15 more
[        ] E/flutter (26675): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.app.AppCompatActivity" on path: DexPathList[[zip file
"/data/app/com.example.meccp-2/base.apk"],nativeLibraryDirectories=[/data/app/com.example.meccp-2/lib/arm64, /data/app/com.example.meccp-2/base.apk!/lib/arm64-v8a, /system/lib64,
/vendor/lib64]]
[   +1 ms] E/flutter (26675):   at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
[   +2 ms] E/flutter (26675):   at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
[        ] E/flutter (26675):   at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
[        ] E/flutter (26675):   ... 15 more
[        ] E/flutter (26675): 
[        ] F/flutter (26675): [FATAL:flutter/shell/platform/android/platform_view_android_jni.cc(77)] Check failed: CheckException(env). 
[        ] F/libc    (26675): Fatal signal 6 (SIGABRT), code -6 in tid 26675 (m.example.meccp)
[  +21 ms] *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
[        ] Build fingerprint: 'Xiaomi/libra/libra:7.0/NRD90M/8.4.28:user/release-keys'
[        ] Revision: '0'
[        ] ABI: 'arm64'
[        ] pid: 26675, tid: 26675, name: m.example.meccp  >>> com.example.meccp <<<
[        ] signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
[   +2 ms] Abort message: '[FATAL:flutter/shell/platform/android/platform_view_android_jni.cc(77)] Check failed: CheckException(env). 
[   +1 ms] '
[        ]     x0   0000000000000000  x1   0000000000006833  x2   0000000000000006  x3   0000000000000008
[        ]     x4   00000000000001e0  x5   8000000000000000  x6   0000007f92bf8000  x7   0000000000000000
[        ]     x8   0000000000000083  x9   ffffffffffffffdf  x10  0000000000000000  x11  0000000000000001
[        ]     x12  ffffffffffffffff  x13  00000000ffffffff  x14  00000000001e0ea0  x15  002b4279d19d3cfe
[        ]     x16  0000007f90086ed8  x17  0000007f900344d0  x18  0000000072218cf0  x19  0000007f92cabb48
[        ]     x20  0000000000000006  x21  0000007f92cabaa0  x22  0000000000000002  x23  0000007f6f195390
[        ]     x24  0000000000000050  x25  0000000000000000  x26  0000007fd75f8fe8  x27  000000000000003d
[        ]     x28  0000000000000001  x29  0000007fd75f8cd0  x30  0000007f90031960
[        ]     sp   0000007fd75f8cb0  pc   0000007f900344d8  pstate 0000000060000000
[ +612 ms] backtrace:
[        ]     #00 pc 000000000006b4d8  /system/lib64/libc.so (tgkill+8)
[   +2 ms]     #01 pc 000000000006895c  /system/lib64/libc.so (pthread_kill+64)
[  +13 ms]     #02 pc 0000000000023ea8  /system/lib64/libc.so (raise+24)
[        ]     #03 pc 000000000001c92c  /system/lib64/libc.so (abort+52)
[        ]     #04 pc 0000000000d04268  /data/app/com.example.meccp-2/lib/arm64/libflutter.so (offset 0xcf0000)
[        ]     #05 pc 0000000000cf7394  /data/app/com.example.meccp-2/lib/arm64/libflutter.so (offset 0xcf0000)
[        ]     #06 pc 0000000000cf58b0  /data/app/com.example.meccp-2/lib/arm64/libflutter.so (offset 0xcf0000)
[        ]     #07 pc 0000000000d397bc  /data/app/com.example.meccp-2/lib/arm64/libflutter.so (offset 0xcf0000)
[        ]     #08 pc 0000000000d04b14  /data/app/com.example.meccp-2/lib/arm64/libflutter.so (offset 0xcf0000)
[        ]     #09 pc 0000000000d08a5c  /data/app/com.example.meccp-2/lib/arm64/libflutter.so (offset 0xcf0000)
[        ]     #10 pc 0000000000018338  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+852)
[        ]     #11 pc 0000000000017f34  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+60)
[        ]     #12 pc 00000000000edca4  /system/lib64/libandroid_runtime.so (_ZN7android18NativeMessageQueue8pollOnceEP7_JNIEnvP8_jobjecti+48)
[        ]     #13 pc 0000000002245fa0  /system/framework/arm64/boot-framework.oat (offset 0x1998000) (android.os.MessageQueue.nativePollOnce+140)
[        ]     #14 pc 0000000002247bb0  /system/framework/arm64/boot-framework.oat (offset 0x1998000) (android.os.MessageQueue.next+236)
[        ]     #15 pc 0000000002241798  /system/framework/arm64/boot-framework.oat (offset 0x1998000) (android.os.Looper.loop+340)
[   +2 ms]     #16 pc 0000000001a56ec4  /system/framework/arm64/boot-framework.oat (offset 0x1998000) (android.app.ActivityThread.main+608)
[        ]     #17 pc 00000000000d1e68  /system/lib64/libart.so (art_quick_invoke_static_stub+600)
[        ]     #18 pc 00000000000de8b0  /system/lib64/libart.so (_ZN3art9ArtMethod6InvokeEPNS_6ThreadEPjjPNS_6JValueEPKc+252)
[        ]     #19 pc 000000000042747c  /system/lib64/libart.so (_ZN3artL18InvokeWithArgArrayERKNS_33ScopedObjectAccessAlreadyRunnableEPNS_9ArtMethodEPNS_8ArgArrayEPNS_6JValueEPKc+108)
[        ]     #20 pc 0000000000428fec  /system/lib64/libart.so (_ZN3art12InvokeMethodERKNS_33ScopedObjectAccessAlreadyRunnableEP8_jobjectS4_S4_m+1188)
[        ]     #21 pc 00000000003a15b4  /system/lib64/libart.so (_ZN3artL13Method_invokeEP7_JNIEnvP8_jobjectS3_S3_+56)
[        ]     #22 pc 00000000006730f8  /system/framework/arm64/boot.oat (offset 0x5a2000) (java.lang.reflect.Method.invoke+180)
[        ]     #23 pc 0000000002916fd0  /system/framework/arm64/boot-framework.oat (offset 0x1998000) (com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run+124)
[        ]     #24 pc 0000000002918284  /system/framework/arm64/boot-framework.oat (offset 0x1998000) (com.android.internal.os.ZygoteInit.main+1472)
[        ]     #25 pc 00000000000d1e68  /system/lib64/libart.so (art_quick_invoke_static_stub+600)
[        ]     #26 pc 00000000000de8b0  /system/lib64/libart.so (_ZN3art9ArtMethod6InvokeEPNS_6ThreadEPjjPNS_6JValueEPKc+252)
[        ]     #27 pc 000000000042747c  /system/lib64/libart.so (_ZN3artL18InvokeWithArgArrayERKNS_33ScopedObjectAccessAlreadyRunnableEPNS_9ArtMethodEPNS_8ArgArrayEPNS_6JValueEPKc+108)
[        ]     #28 pc 00000000004270d4  /system/lib64/libart.so (_ZN3art17InvokeWithVarArgsERKNS_33ScopedObjectAccessAlreadyRunnableEP8_jobjectP10_jmethodIDSt9__va_list+380)
[        ]     #29 pc 00000000003433ac  /system/lib64/libart.so (_ZN3art3JNI21CallStaticVoidMethodVEP7_JNIEnvP7_jclassP10_jmethodIDSt9__va_list+604)
[        ]     #30 pc 000000000009e10c  /system/lib64/libandroid_runtime.so
[        ]     #31 pc 00000000000a0764  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime5startEPKcRKNS_6VectorINS_7String8EEEb+704)
[        ]     #32 pc 000000000000226c  /system/bin/app_process64
[        ]     #33 pc 000000000001a754  /system/lib64/libc.so (__libc_init+88)
[        ]     #34 pc 0000000000001ccc  /system/bin/app_process64
[ +745 ms] Service protocol connection closed.
[   +3 ms] Lost connection to device.
[   +7 ms] DevFS: Deleting filesystem on the device (file:///data/user/0/com.example.meccp/cache/meccpYVENWT/meccp/)
[   +3 ms] Sending to VM service: _deleteDevFS({fsName: meccp})
[ +258 ms] Ignored error while cleaning up DevFS: TimeoutException after 0:00:00.250000: Future not completed
[   +8 ms] "flutter run" took 74,573ms.
[        ] "flutter run" took 74,573ms.

`Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.app.AppCompatActivity"`
you may fixed this problem by add dependency in your build.gradle
`implementation 'com.android.support:appcompat-v7:28.0.0'`

--- a/android/src/main/java/vn/hunghd/flutter/plugins/imagecropper/ImageCropperPlugin.java
+++ b/android/src/main/java/vn/hunghd/flutter/plugins/imagecropper/ImageCropperPlugin.java
@@ -9,8 +9,14 @@ import io.flutter.plugin.common.MethodChannel.Result;
 import io.flutter.plugin.common.PluginRegistry;
 import io.flutter.plugin.common.PluginRegistry.Registrar;
 
+import androidx.appcompat.app.AppCompatDelegate;
+
 /** ImageCropperPlugin */
 public class ImageCropperPlugin implements MethodCallHandler {
+  static
+  {
+    AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
+  }
   private static final String CHANNEL = "plugins.hunghd.vn/image_cropper";
 
   private final Registrar registrar;

